controller/controller.js

```diff
@@ -42,6 +42,29 @@ const getTopics = (request, response) => {
     }).catch(next)
   }
 
+  const postComments = (req, res, next) => {
+    const newComment = req.body;
+    console.log(newComment, "<<<< new comment")
+    insertComment(newComment).then((comment) => {
+        console.log(comment, "<<<<<<<<<<")
+        res.status(201).send({comment})
+    })
+  }
+
+  const deleteCommentById = (req, res, next) => {
+    const { comment_id } = req.params;
+    removeCommentById(comment_id).then((result) => {
+            res.status(204).send();      
+    }).catch(next)
+  };
+
```

Comment:
Nice. An alternative to your response would be `res.sendStatus(204)` which looks cleaner as it doesnt need an empty .send() attached to the end of it